jumpscale/packages/tfgrid_solutions/chats/ubuntu_deploy.py

```diff
@@ -0,0 +1,217 @@
+from jumpscale.god import j
+
+from jumpscale.sals.chatflows.chatflows import GedisChatBot, chatflow_step
+from jumpscale.sals.reservation_chatflow.models import SolutionType
+
+from jumpscale.clients.explorer.models import DiskType
+
+import math
+
+
+class UbuntuDeploy(GedisChatBot):
+    steps = [
+        "ubuntu_start",
+        "ubuntu_network",
+        "ubuntu_name",
+        "ubuntu_version",
+        "container_resources",
+        "container_logs",
+        "public_key_get",
+        "container_node_id",
+        "ubuntu_farm",
+        "container_ip",
+        "expiration_time",
+        "overview",
+        "container_pay",
+        "ubuntu_acess",
+    ]
+
+    @chatflow_step()
+    def ubuntu_start(self):
+        self.user_form_data = dict()
+        self.query = dict()
+        self.HUB_URL = "https://hub.grid.tf/tf-bootable"
```

Comment:
consts should be defined in the top  of module, instance variables should never never depend on the order of invocation